lib/kitchen/provisioner/salt_solo.rb

```diff
@@ -65,7 +65,11 @@ class SaltSolo < Base
         vendor_path: nil,
         vendor_repo: {},
         omnibus_cachier: false,
-        local_salt_root: nil
+        local_salt_root: nil,
+        salt_yum_rpm_key: 'https://repo.saltstack.com/yum/redhat/7/x86_64/archive/%s/SALTSTACK-GPG-KEY.pub',
+        salt_yum_repo: 'https://repo.saltstack.com/yum/redhat/$releasever/$basearch/archive/%s',
+        salt_yum_repo_key: 'https://repo.saltstack.com/yum/redhat/$releasever/$basearch/archive/%s/SALTSTACK-GPG-KEY.pub',
+        salt_yum_repo_latest: 'https://repo.saltstack.com/yum/redhat/salt-repo-latest-2.el7.noarch.rpm'
```

Comment:
This needs some logic to make sure that it grabs the correct rpm for each redhat version.  Not always el7.